frontend/src/lib/arrowProto.ts

```diff
@@ -71,31 +74,33 @@ export interface Styler {
   displayValues?: any
 }
 
-export function toHumanFormat(table: any): DataTable {
+export function parseTable(table: Table): DataFrame {
```

Comment:
Docstrings for all exported functions plz (internal ones are useful as well, of course!)

Reply:
Done